build.gradle

```diff
@@ -17,13 +17,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.3.70'
```

Comment:
Not a big deal, but 1.3.71 is out now and fixes a compiler bug which prevents usage of property delegation in inline functions!